app/models/category.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Category < ApplicationRecord
```

Comment:
Category also needs to have a `cateogry_type` column. Otherwise how will you distinguish between between `food` category and `salary` category?

Reply:
Not addressed.